client/src/helpers/apphelpers.js

```diff
@@ -1,20 +1,21 @@
+const months = {
```

Comment:
If you are going to use more dates, I recommend moment.js. It does all of this for you.

Reply:
OK!